src/liblibc/lib.rs

```diff
@@ -1931,10 +1933,30 @@ pub mod types {
                 pub type PLONG = *mut c_long;
 
                 #[cfg(target_arch = "x86")]
+                pub type LONGLONG = c_double;
```

Comment:
This is a double because of https://msdn.microsoft.com/en-us/library/windows/desktop/aa383751(v=vs.85).aspx

```c++
#if !defined(_M_IX86)
 typedef unsigned __int64 ULONGLONG;
#else
 typedef double ULONGLONG;
#endif
```

```
_M_IX86
```
"Defined for compilations that target x86 processors. This is not defined for x64 processors."

******

Did I read this right? :)